client/extensions/topbar-publish-date/yarn.lock

```diff
@@ -0,0 +1,938 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
you can use yarn for installing if you want, but let's commit only npm lock files

Reply:
Alright, removed it